erpnext/patches/v8_9/remove_employee_from_salary_structure_parent.py

```diff
@@ -0,0 +1,5 @@
+import frappe
+
+def execute():
+	if frappe.get_meta('Salary Structure').has_field('employee'):
```

Comment:
Don't check in meta, instead check in database table.
`if 'employee' in frappe.db.get_table_columns("Salary Structure")`